android/app/src/main/AndroidManifest.xml

```diff
@@ -29,4 +29,10 @@
         </intent-filter>
       </activity>
     </application>
+      <queries>
+        <intent>
+            <action android:name="android.intent.action.VIEW" />
+            <data android:scheme="https"/>
```

Comment:
We don't need mailto/tel at the moment?

Reply:
They were already present for ios. I'd need to ask @SteffiStuffel for that. Therefore, I'll keep it as it is rn.

Reply:
I am pretty sure that that was bc of contact information in the imprint or something and iOS just required to explicitely request permissions earlier than android. So I think that we probably need it as well. But not 100% sure so lets wait for Steffi.

Reply:
Ping @SteffiStuffel

Reply:
Sorry for the late answer. Yes mail and tel is used in the imprint.

Reply:
Those are not pressable atm, created an issue: https://issues.tuerantuer.org/browse/LUN-511